website/docusaurus.config.js

```diff
@@ -71,14 +71,14 @@ var siteSettings = {
     },
     announcementBar: {
       id: "biweekly-demos",
-      content: "Join our bi-weekly demos and see dbt Cloud in action!",
+      content: "Learn about the latest releases and what's coming next to dbt Cloud",
```

Comment:
this looks great! should we have a cta or add a more clear action here?

```suggestion
      content: "Join us on May 14th for the dbt Cloud Launch Showcase event - Discover dbt Cloud's latest innovations and watch live demos! ✨ "
```